bootconfig/README.md

```diff
@@ -0,0 +1,53 @@
+# Bootconfig API
+
+The bootconfig service is responsible for the setting of bootstrap configuration for the device after initial Bootz bootstrap.
+This message may include both the base device configuration but also the new gNSI artifacts.
+
+While this is service which can be used out of band it also allows for the development of an industry standard CLI for recovery
```

Comment:
Industry-standard CLI tool? When I read "industry standard" and "CLI" I think CLI format, which I think isn't what you're proposing.

Reply:
i don't quite follow? i am saying that all vendors should provide a binary which has a standard set of sub commands and args

Reply:
if you mean the cli has consistent args / params for calls and always returns structured data that is exactly what i am saying is a requirement here